testing/citest/tests/kube_smoke_test.py

```diff
@@ -390,47 +395,382 @@ def make_deploy_stage(self, imageSource=None, requisiteStages=[], **kwargs):
+                              'repository': 'nginx'
+                          },
+                          'imagePullPolicy': 'ALWAYS',
+                          'limits': {
+                              'memory': '200Mi'
+                          },
+                          'name': 'nginx',
+                          'requests': {
+                              'cpu': '100m',
+                              'memory': '200Mi'
+                          }}
+                  ],
+                  'kind': 'DaemonSet',
+                  'volumeSources': [],
+                  'provider': 'kubernetes',
+                  'regions': [self.TEST_NAMESPACE],
+                  'region': self.TEST_NAMESPACE
+              }
+          ]
+      }
+      result.update(kwargs)
+      return result
+
+  def make_deploy_stage_statefulset(self, **kwargs):
+      bindings = self.bindings
+      result = {
+          'refId': '1',
+          'type': 'deploy',
+          'name': 'Deploy',
+          'clusters': [
+              {
+                  'account': bindings['SPINNAKER_KUBERNETES_ACCOUNT'],
+                  'application': self.TEST_APP+self.POSTFIX_STATEFUL_DAEMONSET,
+                  'targetSize': 1,
+                  'cloudProvider': 'kubernetes',
+                  'namespace': self.TEST_NAMESPACE,
+                  'containers': [
+                      {
+                          'name': 'Statefulset-Pipeline',
+                          'imageDescription': {
+                              'repository': 'nginx'
+                          },
+                          'imagePullPolicy': 'ALWAYS',
+                          'limits': {
+                              'memory': '200Mi'
+                          },
+                          'name': 'nginx',
+                          'requests': {
+                              'cpu': '100m',
+                              'memory': '200Mi'
+                          }}
+                  ],
+                  'kind': 'StatefulSet',
+                  'volumeSources': [],
+                  'provider': 'kubernetes',
+                  'regions': [self.TEST_NAMESPACE],
+                  'region': self.TEST_NAMESPACE
+              }
+          ]
+      }
+      result.update(kwargs)
+      return result
+
+  def save_delete_statefulset_pipeline(self):
+      deploy_stage = self.make_delete_stage_statefulset()
+      job = dict(
+          appConfig={},
+          keepWaitingPipelines='false',
+          application=self.TEST_APP,
+          name='statefulset-delete-pipeline',
+          lastModifiedBy='anonymous',
+          limitConcurrent='true',
+          parallel='true',
+          stages=[deploy_stage]
+      )
+      payload = self.agent.make_json_payload_from_kwargs(**job)
+      expect_match = {key: jp.EQUIVALENT(value)
+                      for key, value in job.items()}
+      expect_match['stages'] = jp.LIST_MATCHES(
+          [
```

Comment:
This formatting looks off

Reply:
No Its not if you see existing test case 'create_find_image_pipeline', that also has the same